gap/almostsimple.gi

```diff
@@ -677,11 +683,11 @@ RECOG.simplesocle := function(ri,g)
 end;
 
 #! @BeginChunk ComputeSimpleSocle
-#! TODO
+#! This method computes the non-abelian simple socle and stores
+#! it along with additional information if it is called for an
+#! almost simple group and returns <K>NeverApplicable</K>.
```

Comment:
This reads to me as if it always returns `NeverApplicable`, which shouldn't be true, I hope?

Reply:
It always returns `NeverApplicable` as the first time it is invoked it computes the almost simple socle and everytime afterwards the socle is saved and so the function never has to be invoked again. And it also not a "real" recognition success as you are just computing and storing stuff.